gallery/plot_repurposing_annotations.py

```diff
@@ -0,0 +1,68 @@
+"""
+=======================
+Repurposing annotations
+=======================
+
+The following example illustrates the operations available in :ref:`the torchvision.ops module <ops>` for repurposing
+object localization annotations for different tasks (e.g. transforming masks used by instance and panoptic
+segmentation methods into bounding boxes used by object detection methods).
+"""
+
+import matplotlib.patches
+import matplotlib.pyplot
+import numpy
+import skimage.draw
+import skimage.measure
+import torch
+import torchvision.ops.boxes
```

Comment:
In practice shouldn't import the file. (It's correct theoretically, but we should import the only function needed)

We could just do
```suggestion
from torchvision.ops import masks_to_boxes
```